### ISSUE TITLE
conda install not working

### ISSUE THREAD

**Issue by paul-goldsmith** [reymond-group/tmap#18]
I get the following error when attempting the conda install (conda install -c tmap tmap):

Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - tmap

**Comment by Lerbytech** [comment]
I had same error - had to rebuild package from scratch. afair distro doesn't have instructions for conda to build package so I had to check examples for building on other platforms and mcgyver missing stuff. it was not fun and took several evenings without previous experience.

my advice - use clean and new environment to do this. perhaps you will need to downgrade python version

**Comment by Lerbytech** [comment]
I've found instruction and put it as separate issue for future generations. Hope it will help yo https://github.com/reymond-group/tmap/issues/19

**Comment by florian-huber** [comment]
I also did not get it to work with conda, even though trying multiple fresh conda environments with different python versions. The Errors I get are also very inconsitent = confusing.


When I run `conda install -c tmap tmap` from a fresh python 3.7 environment I get  
```
UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - tmap -> python[version='>=2.7,<2.8.0a0|>=3.5,<3.6.0a0|>=3.10,<3.11.0a0']
```

When I run `conda install -c tmap tmap` from a fresh python 3.10 environment I get  
```
UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - tmap -> python[version='>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0']
```

**Comment by florian-huber** [comment]
Sorry, I noticed (too late) that the readme mentions running things under WSL under Windows. 

Using the linux subsystem, the conda install works fine.

**Comment by daenuprobst** [comment]
Yes, It is a suboptimal solution. I got the current version of TMAP running on *some* Windows installs but not on others. I'm currently thinking about writing a small library that only contains the used features of OGDF that is easier to build for all OSs... I have some holidays upcoming and hope to find some time to hack around here :-)
